indico/htdocs/sass/modules/_attachments.scss

```diff
@@ -332,18 +329,17 @@
 
 .material-list {
     .folder .attachment {
-        display: inline-block;
-        overflow: hidden;
+        @include ellipsis;
 
```

Comment:
If I ever added line break after SASS declarations I stopped long ago. I don't think they add any extra clarity when reading SCSS code, nor they are different enough from the rest of the declarations to put them aside.